docs/content/concepts/assets/software-defined-assets.mdx

```diff
@@ -0,0 +1,341 @@
+
+### Loading assets into Dagit
+
+To view and materialize assets in Dagit, you can point it at a module that contains an <PyObject object="AssetGroup" />. E.g.
+
+    dagit -m module_with_asset_group
+
+If you want Dagit to contain both an asset group and a set of [jobs](/concepts/ops-jobs-graphs/jobs-graphs) that target the assets, you can place the <PyObject object="AssetGroup" /> and jobs together inside a [repository](/concepts/repositories-workspaces/repositories).
+
+### Viewing assets in Dagit
+
+Clicking on "Assets", in the right section of Dagit's top navigation pane, takes you to the Asset Catalog, which shows a list of all your assets.
+
+<img
+alt="Asset Catalog"
+src="/images/concepts/assets/software-defined-assets/catalog.png"
+/>
+
+Clicking on the name of one of these assets will take you to the Asset Details Page for that asset.
+
+<img
+alt="Asset Details"
+src="/images/concepts/assets/software-defined-assets/details.png"
+/>
+
+You can view a graph of all the assets with their dependencies by clicking the graph icon to the upper-left of the Asset Catalog, or by clicking "View in Graph" on any of the assets.
+
+<img
+alt="Asset Graph"
+src="/images/concepts/assets/software-defined-assets/graph.png"
+/>
+
+### Materializing assets in Dagit
+
+There are a couple ways in Dagit to launch a run that materializes assets:
+
+- Navigate to the Asset Details Page for the asset and click the "Materialize" button in the upper right corner.
+- In the graph view of the Asset Catalog page, click the "Materialize" button in the upper right corner. You can click on assets to collect a subset to materialize.
+
+## Source assets
+
+It's common for software-defined assets to depend on assets that are generated somewhere else. For example, your data warehouse might contain a set of tables that another team is responsible for ingesting, but that your assets are derived them. To allow the assets in your <PyObject object="AssetGroup" /> to depend on them, you can include <PyObject object="SourceAsset" />s that represent them.
+
+For example:
+
+```python file=/concepts/assets/source_asset.py startafter=start_marker endbefore=end_marker
+from dagster import AssetGroup, AssetKey, SourceAsset, asset
+
+my_source_asset = SourceAsset(key=AssetKey("my_source_asset"))
+
+
+@asset
+def my_derived_asset(my_source_asset):
+    return my_source_asset + [4]
+
+
+asset_group = AssetGroup(assets=[my_derived_asset], source_assets=[my_source_asset])
+```
+
+In any situation where you might use a source asset, you could also have the code inside the asset's op load the data. However, using source assets has a few advantages:
+
+- It allows Dagit to show asset lineage that includes the source assets. If a different <PyObject object="AssetGroup" /> in the same [workspace](/concepts/repositories-workspaces/workspaces) contains an asset definition with the same asset key as a <PyObject object="SourceAsset" />, Dagit can represent the asset lineage across those groups.
```

Comment:
would be nice to link here to the asset lineage page

Reply:
I think the SDA notion of lineage and the notion of lineage on that page are actually a little bit at odds. That page needs a bit of a rework.  Going to leave that out for now to avoid highlighting the rift.